hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -344,13 +344,29 @@ public boolean remove(K key) {
         }
     }
 
+    @Override
+    public boolean invalidate(K key) {
+        try {
+            boolean removed = remove(key);
+            if (removed) {
+                nearCacheStats.incrementInvalidations();
+            }
+            return removed;
+        } finally {
+            nearCacheStats.incrementInvalidationEvents();
```

Comment:
do all calls to this method are result of invalidation events? there can be also _local_ invalidations triggered after calling methods like imap#remove, imap#put ?

Reply:
The method doesn't differ between local or remote invalidations, that's true. But I think that's okay, because that's just due to the actual implementation (we could also have local events, it's just a saved overhead to directly call the invalidate method).

But it will not be increased due to evictions or other internal removals, so it's a pure counter for invalidation requests.

Reply:
can we rename it as `incrementInvalidationRequests` instead of `incrementInvalidationEvents` to make it more clear? Other than this i'm ok with this PR.

Reply:
Done. I've also changed the utility method in `NearCacheTestUtils` accordingly and explained there what an invalidation request includes:
```java
/**
 * Asserts the number of Near Cache invalidation requests (triggered invalidation events and local invalidations).
 *
 * @param context              the given {@link NearCacheTestContext} to retrieve the {@link NearCacheStats} from
 * @param invalidationRequests the given number of Near Cache invalidation requests to wait for
 */
public static void assertNearCacheInvalidationRequests()
```